azadi_pkg.core

```diff
@@ -0,0 +1,29 @@
+CAPI=2:
+name : ::azadi_pkg:1.0
+filesets:
+  risc:
```

Comment:
Check if you can use file set name `azadi` here.

Reply:
Done it.